customize.dist/src/less2/include/alertify.less

```diff
@@ -327,6 +327,8 @@
             nav {
                 padding: @alertify_padding-base;
                 text-align: right;
+                display: flex;
+                align-items: center;
```

Comment:
The level of this change is way too high. All `nav` in modals are affected, check for example the Share modal